### DISCUSSION TITLE
Is unset as a value supporte now?

### DISCUSSION THREAD

**Discussion by zaydek** [facebook/stylex#819] [Q&A] [answered]
I remember in the past seeing errors when I would use "unset" as a value to reset a property. Is this supported now? I'm not seeing any such issue with `0.9.3`.

**Comment by nmn** [comment, marked as answer]
It is supported now, yes. However, in most cases, you should `null` a value instead.

`unset` should only be used to "unset" non-stylex styles.

**Reply by zaydek** [reply]
Right, yeah my use case is just CSS resets not wanting to suppress as style. This is a good tip -- thanks!
